packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -43,9 +44,6 @@ class _MatchesMethodCall extends Matcher {
   }
 }
 
-late TextEditingController controller;
```

Comment:
Let's return these three items to the global scope and dispose them in global tearDown.